cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/PartitionLoadParameters.java

```diff
@@ -97,10 +97,14 @@ protected void initParameters() throws UnsupportedEncodingException {
     _endMs = ParameterUtils.endMs(_request);
     _partitionLowerBoundary = ParameterUtils.partitionBoundary(_request, false);
     _partitionUpperBoundary = ParameterUtils.partitionBoundary(_request, true);
-    _entries = ParameterUtils.entries(_request);
     _minValidPartitionRatio = ParameterUtils.minValidPartitionRatio(_request);
     _allowCapacityEstimation = ParameterUtils.allowCapacityEstimation(_request);
     _brokerIds = ParameterUtils.brokerIds(_request, true);
+    _entries = ParameterUtils.entries(_request);
+    if (_entries < 0) {
+      throw new UserRequestException("Invalid entries value. The entries value must be"
+          + " greater than or equal to 0. Got negative value: " + _entries);
```

Comment:
Nit: Can we make this error message more concise? -- e.g. something like
```
throw new UserRequestException("The requested entries must be positive (Requested: " + _entries + ").");
```

Reply:
Good idea!